packages/services/src/repository/LocalRepository.ts

```diff
@@ -15,7 +15,7 @@ type Configuration =
 const DEFAULT_INDEX_FILENAME = 'index.html';
 const DEFAULT_SERVE_INDEX_ON_NOT_FOUND = false;
 
-export default class LocalRepository implements Repository
+export default class LocalRepository implements ProviderService
```

Comment:
```suggestion
export default class LocalRepository implements Repository
```

I prefer to keep the (empty) `Repository` interface for semantics and consistency reasons.